src/openforms/utils/json_logic.py

```diff
@@ -49,3 +49,123 @@ def normalize_value(value: Union[dict, list, tuple, int, str]) -> JSONLogicValue
         return JsonLogicTest.from_expression(value)
 
     raise NotImplementedError(f"Unknown value type: {type(value)}")
+
+
+def find_json_logic_test_value(value: JsonLogicTest) -> str:
+
+    """returns the value inside a JsonLogicTest"""
+    if isinstance(value, JsonLogicTest) and value.operator == "date":
+        return find_json_logic_test_value(value.values[0])
+    if isinstance(value, JsonLogicTest) and value.operator == "var":
+        return value.values[0]
```

Comment:
Technically I could define some `{"var": {"var": "foo"}}` logic I tink? that would probably break this parsing :grimacing:

Reply:
yes indeed :grimacing: